composer.json

```diff
@@ -16,10 +16,10 @@
     ],
     "autoload": {
         "files": [
-            "src/registration.php"
+            "registration.php"
         ],
         "psr-4": {
-            "ScandiPWA\\CatalogCustomerGraphQl\\": "src"
+            "ScandiPWA\\CatalogCustomerGraphQl\\": ""
         }
```

Comment:
I do not think this PSR-4 mapping is necessary here. Please check if everything works without it (I may be mistaken), and if it does - remove it.

Reply:
Tried to remove if and reinstall it via composer. Now throws issue that can't find requested class (resolver).

Reply:
Okey, then it is indeed necessary.